### PR TITLE
library clean-up: Don't build libNX_Xdmcp anymore. Use system's libXdmcp shared library.

### PR DESCRIPTION
**Pull request by sunweaver**

### CODE REVIEW

File: nx-X11/config/cf/X11.tmpl [hunk 1]
```diff
@@ -2428,6 +2424,7 @@ ProjectUnsharedLibReferences(WINDOWSWM,NX_WindowsWM,$(WINDOWSWMLIBSRC),XBuildLib
 #ifndef ProfileLibXau
 #define ProfileLibXau		NO	/* profiled auth library */
 #endif
+      XDMCPLIBSRC = $(LIBSRC)/Xdmcp
```
Comment (Ionic) [inline nx-X11/config/cf/X11.tmpl]: I guess your editor did something funny here?

This won't work, as the PR explicitly removes $(LIBSRC)/Xdmcp.

File: nx-libs.spec [hunk 2]
```diff
@@ -183,7 +186,7 @@ know a server-specific key called a "magic cookie".
 
 %package -n libNX_Xcomposite1
 Group:          System Environment/Libraries
-Summary:        NX protocol Composite extension client library
+Summary:        NX protocol Composite extension client library (dummy package)
```
Comment (Ionic) [inline nx-libs.spec]: Shouldn't be part of this PR either, probably slipped in.

**Comment by Ionic** [comment]
Looks good otherwise.

File: nx-X11/config/cf/X11.tmpl (hunk 1 above)
Comment (sunweaver) [inline nx-X11/config/cf/X11.tmpl]: On  Mi 22 Apr 2015 18:48:59 CEST, Mihai Moldovan wrote:

> > @@ -2428,6 +2424,7 @@  
> > ProjectUnsharedLibReferences(WINDOWSWM,NX_WindowsWM,$(WINDOWSWMLIBSRC),XBuildLib
> >  #ifndef ProfileLibXau
> >  #define ProfileLibXau       NO  /\* profiled auth library */
> >  #endif
> > -      XDMCPLIBSRC = $(LIBSRC)/Xdmcp
> 
> I guess your editor did something funny here?
> 
> This won't work, as the PR explicitly removes $(LIBSRC)/Xdmcp.

Ooopsss... some copy+paste thingy...

Fixed with next force-push...

## Mike

DAS-NETZWERKTEAM
mike gabriel, herweg 7, 24357 fleckeby
fon: +49 (1520) 1976 148

GnuPG Key ID 0x25771B31
mail: mike.gabriel@das-netzwerkteam.de, http://das-netzwerkteam.de

freeBusy:
https://mail.das-netzwerkteam.de/freebusy/m.gabriel%40das-netzwerkteam.de.xfb

**Comment by sunweaver** [comment]
On  Mi 22 Apr 2015 19:02:28 CEST, Mihai Moldovan wrote:

> Looks good otherwise.

OK. Thanks again! Force-pushed and rebased against 3.6.x. Please merge now.

Thanks,

## Mike

DAS-NETZWERKTEAM
mike gabriel, herweg 7, 24357 fleckeby
fon: +49 (1520) 1976 148

GnuPG Key ID 0x25771B31
mail: mike.gabriel@das-netzwerkteam.de, http://das-netzwerkteam.de

freeBusy:
https://mail.das-netzwerkteam.de/freebusy/m.gabriel%40das-netzwerkteam.de.xfb
